components/user/UserLink.js

```diff
@@ -2,6 +2,17 @@ import colors from "@config/icons.json";
 import getIcon from "@components/Icon";
 import Link from "@components/Link";
 import Edit from "@components/account/manage/Edit";
+import styles from '../../styles/animation.module.css';
+
+const getLinkAnimation = new Map([
+  ["Pulse", "group animate-pulse"],
+  ["Bounce", "animate-bounce opacity-75"],
+  ["Glow", `${styles.glow}`],
```

Comment:
I think this can be the variable directly

```suggestion
  ["Glow", styles.glow],
```